plugins/inputs/fritzbox/fritzbox_test.go

```diff
@@ -0,0 +1,167 @@
+package fritzbox
+
+import (
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/parsers/influx"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+	"github.com/tdrn-org/go-tr064/mock"
```

Comment:
Please group the libs.

Reply:
Adapted via 1ada7a8c36211e8f74afa86ed9638c45083686bb